src/typecheck/operation.rs

```diff
@@ -195,6 +195,8 @@ pub fn get_uop_type(
                 ("groups", mk_typewrapper::array(AbsType::Str()))
             ),
         ),
+        // Dyn -> Dyn
+        UnaryOp::Force(_) => (mk_typewrapper::dynamic(), mk_typewrapper::dynamic()),
```

Comment:
Why does this appear in this PR?

Reply:
Because we merged in `feature/force-operator`?